frontend/src/metabase/collections/containers/CollectionSidebar.jsx

```diff
@@ -41,16 +42,17 @@ const Sidebar = styled(Box)`
     we should eventually refactor code elsewhere in the app to use this by default instead of determining the relationships clientside, but this works in the interim
   */
   query: () => ({ tree: true }),
+  loadingAndErrorWrapper: false,
```

Comment:
I know in our style guide we want to try and discourage too many unnecessary comments but this feels like a place where having a note about why we're setting this to false would be likely be useful in the future.